eth_utils/hexadecimal.py

```diff
@@ -45,7 +46,39 @@ def add_0x_prefix(value: HexStr) -> HexStr:
     return HexStr("0x" + value)
 
 
+def is_hexstr(value: Any) -> bool:
+    if not isinstance(value, str):
```

Comment:
Any reason you decided to go with `isinstance()` here besides `is_text()` (like in `is_hex()`)? It seems to me like they're doing essentially the same thing, so you might consider going with `is_text()` for consistency.